### ISSUE TITLE
Compiler_builtins - LLVM ERROR: Cannot select

### ISSUE THREAD

**Issue by Freddyfux** [rust-lang/compiler-builtins#445]
I want to use msp430-quickstart and got this compiler-builtins error:
```
PS D:\Development\Rust\test_rust\msp430-test> cargo build --target msp430-none-elf -Z build-std=core        
warning: unused manifest key: package.msp430
warning: unused manifest key: package.msp430g2553
   Compiling compiler_builtins v0.1.53
   Compiling bare-metal v0.2.5
   Compiling vcell v0.1.3
   Compiling msp430 v0.2.2
   Compiling msp430-rt v0.2.5
   Compiling panic-msp430 v0.2.0
   Compiling msp430g2553 v0.2.0
LLVM ERROR: Cannot select: 0x20571ef7f58: i16,ch = AtomicLoad<(load unordered (s16) from %ir.188)> 0x20570988590:1, 0x20570988590, C:\Users\Martin\.cargo\registry\src\github.com-1ecc6299db9ec823\compiler_builtins-0.1.53\src\mem\impls.rs:65:29 @[ C:\Users\Martin\.cargo\registry\src\github.com-1ecc6299db9ec823\compiler_builtins-0.1.53\src\mem\impls.rs:109:13 @[ C:\Users\Martin\.cargo\registry\src\github.com-1ecc6299db9ec823\compiler_builtins-0.1.53\src\mem\mod.rs:25:5 ] ]
  0x20570988590: i16,ch = load<(dereferenceable load (s16) from %ir.28)> 0x20571ef3c40, FrameIndex:i16<24>, 
undef:i16, C:\Users\Martin\.cargo\registry\src\github.com-1ecc6299db9ec823\compiler_builtins-0.1.53\src\mem\impls.rs:65:69 @[ C:\Users\Martin\.cargo\registry\src\github.com-1ecc6299db9ec823\compiler_builtins-0.1.53\src\mem\impls.rs:109:13 @[ C:\Users\Martin\.cargo\registry\src\github.com-1ecc6299db9ec823\compiler_builtins-0.1.53\src\mem\mod.rs:25:5 ] ]
    0x20571ef65b8: i16 = FrameIndex<24>
    0x20570993a50: i16 = undef
In function: memcpy
error: could not compile `compiler_builtins`
warning: build failed, waiting for other jobs to finish...
error: build failed
```
I have no clue what this means and how to fix it. Maybe someone can help me. I am working with a Windows engine.
[log.txt](https://github.com/rust-lang/compiler-builtins/files/7644737/log.txt)

**Comment by bjorn3** [comment]
Duplicate of https://github.com/rust-lang/compiler-builtins/issues/441. It is fixed in compiler-builtins 0.1.55. I am not sure if rustc has already updated to it. You can try updating rustc.

**Comment by cr1901** [comment]
@Freddyfux There was no way to work around the "cannot select" without fixing `compiler-builtins`. So I didn't update the quickstart, knowing I would get a fix in semi-shortly, and hoped nobody else would notice :P. Apparently not...
